app/src/main/res/navigation/add_navigation.xml

```diff
@@ -20,5 +20,9 @@
         android:id="@+id/category_bottom_sheet_dialog"
         android:name="com.gholem.moneylab.features.add.BottomSheetCategoryFragment"
         android:label="CategoryBottomSheet"
-        tools:layout="@layout/bottomsheet_category_fragment" />
+        tools:layout="@layout/bottomsheet_category_fragment" >
+        <action
+            android:id="@+id/action_category_bottom_sheet_dialog_to_add_fragment"
```

Comment:
You are just navigating up from this fragment so you don't need action - it's unused.